### ISSUE TITLE
Add baselines to real-time eval

### ISSUE THREAD

**Issue by dylanhmorris** [CDCgov/pyrenew-hew#183]
This will enable relative WIS calculations, etc.

Simplest approach might be to generate a hubverse table with all models (including baselines)

**Comment by damonbayer** [comment]
Yes, I would like to use the baseline as a fallback forecast, so anything we are doing with the real forecasts (like making plots) should also be done for baseline.

**Comment by dylanhmorris** [comment]
This has been completed by the porting of real-time eval to the minihub
